subprojects/architecture-test/build.gradle.kts

```diff
@@ -65,6 +67,24 @@ tasks.test {
         // PTS doesn't work well with architecture tests which scan all classes
         enabled = false
     }
+
+    doLast {
+        resortStoredRules(ruleStoreDir)
```

Comment:
❓ Don't we have to register `ruleStoreDir` as an output, since we're now (potentially) changing it in the task?

Reply:
I think I've understood what were you mentioning, PTAL!

Reply:
That's it, looks good.